Jenkinsfile

```diff
@@ -13,10 +13,22 @@ pipeline {
     }
     stages {
         stage('Test') {
+            environment {
+                CREDS_FILE = credentials('pipeline-e2e-creds')
+                LOGDNA_HOST = "logs.use.stage.logdna.net"
```

Comment:
Not sure we want to consider the staging environment stable enough to use for reliable testing. We can isolate the agent completely and use a mock ingester  that tracks what lines have come through. I believe we already have a yaml for that

Reply:
I was going to build that functionality into a future version of the test binary, I didn't know we had a mock ingester.

I used staging as that's what Test suggested. I'm not _too_ concerned about the investees being available in staging, but I agree having something capture the lines and validate delivery would be better here.

I would still like to test that we successfully write to the real ingesters somewhere, maybeit should like in a separate test stage that doesn't automatically fail the build, or checks the api is available and skips if it isn't?